client/src/api/block_builder/mod.rs

```diff
@@ -336,47 +336,6 @@ impl<'a> ClientBlockBuilder<'a> {
         }
     }
 
-    /// Get output amount and address from an OutputDto, governance_transition for Alias Outputs so we get the unlock
-    /// condition we're interested in
-    pub fn get_output_amount_and_address(
```

Comment:
Can you elaborate why it's been removed ?

Reply:
Because we have better alternatives :trollface: 
It was a leftover from Chrysalis